linked_list/linked_list.py

```diff
@@ -0,0 +1,55 @@
+from node import Node
+
+
+class LinkedList:
+    def __init__(self):
+        self.first_node = None
+        self.last_node = None
+        self.length_of_list = 0
+
+    def append(self, new_value):
+        """Add a new element at the end of the list."""
+        if new_value is None:
+            raise TypeError("Value is absent!")
+        else:
+            self.length_of_list += 1
+            if self.first_node is None:
+                self.first_node = Node(new_value)
+                self.last_node = self.first_node
+            else:
+                self.last_node.next_node = Node(new_value)
+
+    def get_size(self):
+        """Return size of the list."""
+        self.length_of_list = 0
+        if self.first_node is not None:
+            self.length_of_list += 1
+            current_node = self.first_node
+            while current_node.next_node is not None:
+                current_node = current_node.next_node
+                self.length_of_list += 1
```

Comment:
Because it's so inefficient to calculate the size - it's better to have another field in the class for it. And increment it in case of `append`, and decrement it in `delete` method.

Reply:
Oh, I just found the `length_of_list` field - why not just return it in this method? That's the field that I was talking about.